web/src/narrow_banner.ts

```diff
@@ -124,12 +124,28 @@ export function pick_empty_narrow_banner(): NarrowBannerData {
                   },
               ),
     };
+    const combined_feed_banner = {
+        title: $t({defaultMessage: "There are no messages in your combined feed."}),
+        // Spectators cannot start a conversation.
```

Comment:
This comment doesn't make sense in this context, but you are correct in that we should not show the public channels note for spectators since that isn't a useful filter for the logged out view.